README.md

```diff
@@ -61,8 +61,11 @@ This is a react diseases project that renders the covid updates from different c
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo Link](https://www.loom.com/share/bf8adf1ace3d4b3caf857bad0e366b46)
+
+## Live Video Link
+- [Live Video Link](https://www.loom.com/share/bf8adf1ace3d4b3caf857bad0e366b46)
 
-- [Live Demo Link]()
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [OPTIONAL] Great job making the changes requested by the previous reviewer 👏🏻 But, it would also be better to correct the `issues page` link from the "Contributing" section to the correct page 🧑‍🔧